ietf/meeting/management/commands/import_iab_agendas.py

```diff
@@ -0,0 +1,240 @@
+        "2022-04-20.md",
+        "2022-04-27.md",
+        "2022-05-04.md",
+        "2022-05-11.md",
+        "2022-06-01.md",
+        "2022-06-15.md",
+        "2022-06-22.md",
+        "2022-06-29.md",
+        "2022-07-06.md",
+        "2022-07-24.md",
+        "2022-07-26.md",
+        "2022-08-10.md",
+        "2022-08-24.md",
+        "2022-09-07.md",
+        "2022-09-21.md",
+        "2022-09-28.md",
+        "2022-10-05.md",
+        "2022-10-12.md",
+        "2022-10-26.md",
+        "2022-11-06.md",
+        "2022-11-08.md",
+        "2022-11-10.md",
+        "2022-11-23.md",
+        "2022-12-07.md",
+        "2022-12-14.md",
+    ]
+
+
+class Command(BaseCommand):
+    help = "Performs a one-time import of older IAB agendas"
+
+    def handle(self, *args, **options):
+        if Document.objects.filter(name="agenda-interim-2018-iab-26-20180905").exists():
+            print("Command has already been run - exiting")
+            exit(0)
+
+        tmpdir = tempfile.mkdtemp()
+        process = subprocess.Popen(
+            ["git", "clone", "https://github.com/kesara/iab-scraper.git", tmpdir],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        stdout, stderr = process.communicate()
+        if not Path(tmpdir).joinpath("iab_agendas", "2018-09-05.md").exists():
+            print("Git clone of the iab-scraper directory did not go as expected")
+            print("stdout:", stdout)
+            print("stderr:", stderr)
+            print(f"Clean up {tmpdir} manually")
+            exit(-1)
+
+        agendas = agendas_to_import()
+        for agenda in agendas:
+            [year, month, day] = [int(part) for part in agenda[:10].split("-")]
+            agenda_date = datetime.date(year, month, day)
+            meeting = Meeting.objects.get(
+                date=agenda_date, type_id="interim", session__group__acronym="iab"
+            )
+            counter = int(meeting.number.split("-")[3])
+            agenda_docname = (
+                f"agenda-interim-{year}-iab-{counter:02d}-{agenda_date:%Y%m%d}"
+            )
+            agenda_filename = f"{agenda_docname}-00.md"
+            # Create Document
+            doc = Document.objects.create(
+                name=agenda_docname,
+                type_id="agenda",
+                title=f"Agenda {meeting.number} {agenda_date:%Y-%m-%d}",
+                group_id=7,  # The IAB group
+                rev="00",
+                uploaded_filename=agenda_filename,
+            )
+            DocAlias.objects.create(name=doc.name).docs.add(doc)
+            e = DocEvent.objects.create(
+                type="comment",
+                doc=doc,
+                rev="00",
+                by_id=1,  # The "(System)" person
+                desc="Agenda moved into datatracker from iab wordpress website",
+            )
+            doc.save_with_history([e])
+
+            session = Session.objects.get(meeting=meeting)
```

Comment:
If this were more than a one-off, I'd suggest handling possible failure of `get()` here. However, since it's not, I don't.